.github/workflows/test.yml

```diff
@@ -56,13 +56,6 @@ jobs:
           GRAFANA_CLOUD_USER_ID: "356840"
           GRAFANA_CLOUD_URL: "https://logs-prod-017.grafana.net"
           GRAFANA_CLOUD_API_KEY: "glc_eyJvIjoiNzcwNzY0IiwibiI6Im1ldHJpY3Nsb2dzdHJhY2Vzd3JpdGVyLWdpdGh1Yi1kYWdnZXItZGFnZ2VyLXJlcG8tYW5kLWZvcmtzIiwiayI6Ijk4My1MX3M0fHpdLDMjNHo/MyswKyg1MCIsIm0iOnsiciI6InVzIn19"
-      - name: PR comment
-        # Only comment on PRs
-        if: github.event_name == 'pull_request'
-        uses: thollander/actions-comment-pull-request@v2
-        with:
-          filePath: ./comment.md
```

Comment:
What should we do about the generated `./comment.md` file in the previous step.
Also, shouldn't the keys/token exposed in it be put in secrets instead?

Reply:
Found my answer [here](https://github.com/dagger/dagger/pull/4340/commits/f4db2ed45602137b467af21e1b17890a10a063b0#r1086878647)

Reply:
> What should we do about the generated ./comment.md file in the previous step.

Nothing for now. We still want this functionality, but since it's not as straightforward with forks, based on everything I read, this seems to be the quickest fix to unblock everyone else.

> Also, shouldn't the keys/token exposed in it be put in secrets instead?

No, because then forks will not have access to them. Those tokens have been scoped to metrics, logs & traces write access only, so the worst that can happen is someone sending garbage data. I don't think that will be worth the effort. If we end up in that situation, I am happy to cross that bridge when we get there, but for now this is good enough.